src/all/misskon/src/eu/kanade/tachiyomi/extension/all/misskon/MissKon.kt

```diff
@@ -0,0 +1,258 @@
+package eu.kanade.tachiyomi.extension.all.misskon
+
+import eu.kanade.tachiyomi.lib.randomua.UserAgentType
+import eu.kanade.tachiyomi.lib.randomua.setRandomUserAgent
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimitHost
+import eu.kanade.tachiyomi.source.model.Filter
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.util.asJsoup
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.concurrent.TimeUnit
+
+class MissKon() : SimpleParsedHttpSource() {
+
+    override val baseUrl = "https://misskon.com"
+    override val lang = "all"
+    override val name = "MissKon"
+    override val supportsLatest = true
+
+    override val client = network.cloudflareClient.newBuilder()
+        .rateLimitHost(baseUrl.toHttpUrl(), 10, 1, TimeUnit.SECONDS)
+        .setRandomUserAgent(UserAgentType.MOBILE)
+        .build()
+
+    override fun simpleMangaSelector() = "article.item-list"
+    override fun simpleMangaFromElement(element: Element): SManga {
+        val titleEL = element.select(".post-box-title")
+
+        val manga = SManga.create()
+        manga.title = titleEL.text()
+        manga.thumbnail_url = element.select(".post-thumbnail img").attr("data-src")
+        manga.setUrlWithoutDomain(titleEL.select("a").attr("href").substring(baseUrl.length))
```

Comment:
Don't need `.substring`, `setUrlWithoutDomain` does this for you. You should in addition use `.absUrl` so the responsibility of catering for relative URLs, weird schema syntax etc. is handled by Jsoup rather than `setUrlWithoutDomain` -- which does not have any additional context.